gateway/utils/syncman/syncman.go

```diff
@@ -32,6 +35,11 @@ type Manager struct {
 
 	// For authentication
 	adminMan *admin.Manager
+
+	// Modules
```

Comment:
Remove cb from the struct

Reply:
Some functions still call s.cb so should I call SetProjectConfig instead at those places?

Reply:
Which functions?